### PR TITLE
Add failing test to reproduce #30.

### PR DESCRIPTION
**Pull request by tanner0101**

### CODE REVIEW

**Comment by tanner0101** [comment]
@ncvitak added you to the team. until then, this is an informal request to review :)

File: Sources/JWT/JWT.swift [hunk 1]
```diff
@@ -12,6 +12,14 @@ public struct JWT {
     public let payload: Node
     public let signature: String
 
+    /// Used to store the token that created this
+    /// JWT if it was parsed
+    public let rawToken: (
```
Comment (vzsg) [inline Sources/JWT/JWT.swift]: I don't think it's necessary to expose this to `public` scope.

**Comment by ncvitak** [comment]
Everything looks good to me. I'm just having trouble building it so I can test.

Specifically the CTLS module from Crypto isn't building for me

**Comment by tanner0101** [comment]
@ncvitak
```
brew install openssl 
```
Then either 
```
export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
swift build 
```

Or 

```
swift build -Xswiftc -I/usr/local/opt/openssl/include -Xlinker -L/usr/local/opt/openssl/lib
```

**Pull request merged**